scene/resources/shader.cpp

```diff
@@ -136,9 +150,38 @@ void Shader::get_shader_uniform_list(List<PropertyInfo> *p_params, bool p_get_gr
 			if (pi.type == Variant::RID) {
 				pi.type = Variant::OBJECT;
 			}
+#ifdef TOOLS_ENABLED
+			if (Engine::get_singleton()->is_editor_hint()) {
+#ifdef MODULE_REGEX_ENABLED
+				const RegEx pattern("/\\*\\*([^*]|[\\r\\n]|(\\*+([^*/]|[\\r\\n])))*\\*+/\\s*uniform\\s+\\w+\\s+" + pi.name + "(?=[\\s:;=])");
+				Ref<RegExMatch> pattern_ref = pattern.search(code);
+				if (pattern_ref != nullptr) {
+					RegExMatch *match = pattern_ref.ptr();
+					const RegEx pattern_tip("\\/\\*\\*([\\s\\S]*?)\\*/");
+					Ref<RegExMatch> pattern_tip_ref = pattern_tip.search(match->get_string(0));
+					RegExMatch *match_tip = pattern_tip_ref.ptr();
+					const RegEx pattern_stripped("\\n\\s*\\*\\s*");
+					DocData::PropertyDoc prop_doc;
+					prop_doc.name = "shader_parameter/" + pi.name;
+					prop_doc.description = pattern_stripped.sub(match_tip->get_string(1), "\n", true);
+					class_doc.properties.push_back(prop_doc);
+				}
+#else
+				DocData::PropertyDoc prop_doc;
+				prop_doc.name = "shader_parameter/" + pi.name;
+				// prop_doc.description = "(Regex module is not enabled, shader parameter documentation will not be available.)";
+				class_doc.properties.push_back(prop_doc);
+#endif
+			}
+#endif
 			p_params->push_back(pi);
 		}
 	}
+#ifdef TOOLS_ENABLED
+	if (!class_doc.name.is_empty() && p_params) {
+		EditorHelp::get_doc_data()->add_doc(class_doc);
```

Comment:
this line causes https://github.com/godotengine/godot/issues/91306

maybe add a "Engine::get_singleton()->is_editor_hint()" in the if check right above it? that fixed it for me at least

Reply:
Thanks for the report, #91307 should fix it.